src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -105,7 +105,11 @@ const ReservationList: FC<ReservationListProps> = ({ pageSize }) => {
         <h1 className="text-header-h1 m-32">
           {t("reservationListHeaderText")}
         </h1>
-        {isLoadingUserData && <div className="ssc-square w-90 ml-32 my-32" />}
+        {isLoadingUserData && (
+          <div className="ssc">
+            <div className="ssc-square w-90 ml-32 my-32" />
```

Comment:
Could be a small separate component

Reply:
Wrote a comment that it's a skeleton version of <ReservationPauseToggler />